controllers/replication-controller/dellcsireplicationgroup_controller.go

```diff
@@ -773,6 +782,34 @@ func removePVClaimRef(ctx context.Context, client connection.RemoteClusterClient
 	return fmt.Errorf("timed out waiting on Local PV Claim Ref to be removed")
 }
 
+func removeReservedClaimRefforTargetPV(ctx context.Context, client connection.RemoteClusterClient, pvName string, log logr.Logger) error {
```

Comment:
Update the function name with proper camel case 
removeReservedClaimRefforTargetPV() => removeReservedClaimRefForTargetPV()
that is for => For